qa/tasks/scripts/ses_tuned.sh

```diff
@@ -0,0 +1,41 @@
+set -ex
+
+check_tuned(){
+role=$1
+
+if [ $(echo "$roles_list" | wc -l) -gt 1 ]
+then
+    for server in $(eval $getservers | awk '{print $2}')
+    do 
+        role=$(salt $server cmd.run "tuned-adm active | egrep '$role|virtual-guest'" --output=json | jq -r .[] | cut -d : -f 2)
+    done
+fi 
+  
+salt -L $(eval $getservers | awk '{print $2}' | tr '\n' ',') cmd.run "tuned-adm active | egrep '$role|virtual-guest'"
```

Comment:
Add indentation to function body, and the same as below:
 avoid extra salt `get minions` call and non-necessary 'eval'.
it is more reasonable to pass minions as an array to the check_tuned function, like:
```
function check_tuned() {
    role=$1
    shift
    minions="$@" # this renders rest of arguments as single space separated list
    roles=($role_list)
    if [ ${#roles} -gt 1 ] ; then
        for server in $minions ; do
            role=$(salt $server cmd.run "tuned-adm active | egrep '$role|virtual-guest'" --output=json | jq -r .[] | cut -d : -f 2)
      done
    fi
    salt -L ${minions// /,} cmd.run "tuned-adm active | egrep '$role|virtual-guest'" 
}
```
and later use
```
check_tuned $role ${minions[@]}
```